classes/village/VillageApiPresenter.php

```diff
@@ -161,6 +161,84 @@ function (WarLogDto $war_log) {
             "player_war_log" => $player_war_log
         ];
     }
+    public static function warRecordDataResponse(System $system, User $player, int $page_number = 1): array
```

Comment:
- This returns multiple war records so we could make it plural
- everything in these responses is kinda "data" so maybe we can drop that word from the naming
```suggestion
    public static function warRecordsResponse(System $system, User $player, int $page_number = 1): array
```

Reply:
Keeping it 1:1 with the name used in the template file and js, explanation for specifying "data" in the other comment.